encoders/image/TirgImageEncoder/__init__.py

```diff
@@ -0,0 +1,63 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import sys
+import pickle
+
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.encoders.frameworks import BaseTorchEncoder
+from jina.excepts import PretrainedModelFileDoesNotExist
+
+sys.path.append(".")
+from img_text_composition_models import TIRG
```

Comment:
Hide these import inside functions, and I think we can skip the `sys path` (u can see VSEImageEncoder for reference)